content/microcontrollers/pca10031.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "PCA10031"
+weight: 3
+---
+
+The [Nordic Semiconductor PCA10031](https://www.nordicsemi.com/eng/Products/nRF51-Dongle) is a low-cost, versatile USB development dongle for wireless applications based on the Nordic Semiconductor [nRF51422](https://www.nordicsemi.com/eng/Products/ANT/nRF51422) chip using the nRF51 series SoC.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | ? | No |
+| I2C      | ? | No |
+| ADC      | ? | No |
+| PWM      | ? | No |
```

Comment:
The nrf51822 and nrf51422 are almost identical (the difference is support for certain wireless protocols). I think it is safe to assume the support is equivalent to nrf51822.

Reply:
Updated.